test/test.js

```diff
@@ -76,20 +76,26 @@ test('skip throwing when using reject option in sync mode', t => {
 
 const testStripFinalNewline = async (t, index, stripFinalNewline, execaCommand) => {
 	const {stdio} = await execaCommand('noop-fd.js', [`${index}`, 'foobar\n'], {...fullStdio, stripFinalNewline});
-	t.is(stdio[index], `foobar${stripFinalNewline ? '' : '\n'}`);
+	t.is(stdio[index], `foobar${stripFinalNewline === false ? '\n' : ''}`);
 };
 
-test('stripFinalNewline: true with stdout', testStripFinalNewline, 1, undefined, execa);
+test('stripFinalNewline: undefined with stdout', testStripFinalNewline, 1, undefined, execa);
+test('stripFinalNewline: true with stdout', testStripFinalNewline, 1, true, execa);
```

Comment:
Default value is `true`, so this tests that default values are correctly assigned.